hikari/audit_logs.py

```diff
@@ -47,9 +47,14 @@
 import attr
 
 from hikari import snowflakes
+from hikari.channels import GuildTextChannel
```

Comment:
We rather not have singular imports as it is not the code style we are going for.

These should be changed to `from hikari import channels` and then access it using `channels.GuildTextChannel`. Also, if these are for typehinting, please move them under `if typing.TYPE_CHECKING`

Edit:
I just realized `from hikari import channels` is already under `if typing.TYPE_CHECKING`, so you can just remove these and use what I said above

Reply:
aight

Reply:
but I also use them in `assert isinstance(channel, TextChannel)`

Reply:
In this cause you need to move `from hikari import channels` out of `if typing.TYPE_CHECKING:` and into the root and then use `channels.TextChannel`